jackson-databind/src/main/java/com/fasterxml/jackson/databind/deser/impl/CreatorCandidate.java

```diff
@@ -22,17 +22,17 @@ protected CreatorCandidate(AnnotationIntrospector intr,
         _paramCount = count;
     }
 
-        public static CreatorCandidate construct(AnnotationIntrospector intr,
-                AnnotatedWithParams creator, BeanPropertyDefinition[] propDefs)
-        {
-            final int pcount = creator.getParameterCount();
-            Param[] params = new Param[pcount];
-            for (int i = 0; i < pcount; ++i) {
-                AnnotatedParameter annParam = creator.getParameter(i);
-                JacksonInject.Value injectId = intr.findInjectableValue(annParam);
-                params[i] = new Param(annParam, (propDefs == null) ? null : propDefs[i], injectId);
-            }
-    return new CreatorCandidate(intr, creator, params, pcount);    }
+    public static CreatorCandidate construct(AnnotationIntrospector intr,
+                AnnotatedWithParams creator, BeanPropertyDefinition[] propDefs) {
+        final int pcount = creator.getParameterCount();
+        Param[] params = new Param[pcount];
+        for (int i = 0; i < pcount; ++i) {
+            AnnotatedParameter annParam = creator.getParameter(i);
+            JacksonInject.Value injectId = intr.findInjectableValue(annParam);
+            params[i] = new Param(annParam, (propDefs == null) ? null : propDefs[i], injectId);
+        }
+        return new CreatorCandidate(intr, creator, params, pcount);    
```

Comment:
Please remove trailing spaces and fix merge conflicts

Reply:
@que-liu please address this in 2-3 days or we'll assign to someone else.